index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Document</title>
+  <link rel="stylesheet" href="css/style.css" />
+  <script src="https://kit.fontawesome.com/5784b9ea6f.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+  <div class="wrapper">
+    <header>
+      <nav class="flex-nav">
+        <h1 class="logo"><a href="#">My Logo</a></h1>
+        <div class="toggleNav"><i class="fa fa-bars"></i></div>
+        <ul>
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+      </nav>
+    </header>
+    <section>
+      <h1>I&rsquo;m Francis.<br />Glad to see you!</h1>
+      <p>
+        I&rsquo;m a software developer! I can help you build a product ,
+        feature or website Look through some of my work and experience! If you
+        like what you see and have a project you need coded, don&rsquo;t
+        hesitate to contact me.
+      </p>
+      <a href="#" class="connect">LET’S CONNECT</a>
```

Comment:
Kindly ensure to make use of the right font family and font weight for your texts so as to make them match the given design 😉.

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img width="350" alt="image" src="https://github.com/lukademinnosa/module1_project1/assets/64643995/16826495-1343-4f60-ac84-04fe55bff26f">|<img width="350" alt="image" src="https://github.com/lukademinnosa/module1_project1/assets/64643995/15bcc6c1-7bdc-47b5-893e-0a9c362e4e43">|